src/js/components/FilterByTaskState.js

```diff
@@ -79,14 +76,20 @@ FilterByTaskState.propTypes = {
   currentStatus: React.PropTypes.string,
   handleFilterChange: React.PropTypes.func,
   statuses: React.PropTypes.array.isRequired,
-  totalTasksCount: React.PropTypes.number.isRequired
+  totalTasksCount: React.PropTypes.number.isRequired,
+
```

Comment:
often (but especially reactjs-components) we like to separate classnames into a separate section so it's easy to spot out the classnames props vs the other props